.gitignore

```diff
@@ -0,0 +1,6 @@
+# Executable names
+rps
+borgor
+
+# Database
+*.db
```

Comment:
Btw I added *.db to .gitignore because I plan to standardise the package location so it's not relative to rps executable anymore.